frontend/talentsearch/src/js/components/pool/PoolAdvertisementPage.tsx

```diff
@@ -78,10 +78,12 @@ const IconTitle = ({ children, icon }: IconTitleProps) => {
   );
 };
 
+const recruitmentEmailAddress = "recruitmentimit-recrutementgiti@tbs-sct.gc.ca";
```

Comment:
Could this use `TALENTSEARCH_RECRUITMENT_EMAIL` from talentSearchConstants.tsx instead of a static value?

Reply:
Doh, I was the one who added that comment to the issue!  Good catch.